### DISCUSSION TITLE
How to remove redirect after submitted custom event

### DISCUSSION THREAD

**Discussion by achimonchi** [plausible/analytics#2616] [Self-Hosted Support]
Hello, i want to know how to remove redirect form.
im using `nextjs`, and my code is..
```js
function submit(e:FormEvent){
  e.preventDefault()
  // plausible will refresh to this page again, not direct to /home
  Router.push('/home')
}

return(
  <form onSubmit={submit} className="plausible-event-name=Signin"></form>
)
```
when user submit, my apps will send event to my plausible self hosted, than refresh the page..
how to remove callback plausible to refresh the page?

**Comment by jthompson22** [comment]
@achimonchi  did you solve this? I am having the same problem
